scripts/create-test-data.js

```diff
@@ -26,30 +26,29 @@ const utf16CharCodesToString = (typedArray) => {
   return val;
 };
 
+const getManifests = () => (fs.readdirSync(manifestsDir) || [])
```

Comment:
This file is slightly refactored so that we can get a list of manifests and segments in two different places.